eng/packages/General.props

```diff
@@ -51,7 +51,6 @@
     <PackageVersion Include="System.IO.Hashing" Version="$(SystemIOHashingVersion)" />
     <PackageVersion Include="System.Memory" Version="4.5.5" />
     <PackageVersion Include="System.Net.Http.Json" Version="$(SystemNetHttpJsonVersion)" />
-    <PackageVersion Include="System.Net.Http" Version="4.3.4" />
```

Comment:
While I'm fine with the change, why did you singled out System.Net.Http here? For instance, while validating this change I noticed that System.Net.Http.Json is also present on this file and also not used apparently. One final thing to check, is I think that CPM will also affect transitive dependencies, so it would be good to double check and see if these aren't needed there.

Reply:
The issue is about System.Net.Http, so I only looked at System.Net.Http. I didn't look at anything else as it wasn't in the scope of this change.

Reply:
I see. Can we either cleanup the rest of unused ones here or if not then log an issue to cleanup to track the follow up?

Reply:
I'm going to send a separate PR for the cleanup.